ntbs-service/Pages/Notifications/Overview.cshtml

```diff
@@ -9,6 +10,13 @@
     ViewData["CurrentPage"] = NotificationSubPaths.Overview;
 }
 
+@if (Model.Notification.NotificationStatus == NotificationStatus.Closed)
+{
+    <div class="notification-overview-details-container">
+        <h5>This notification is closed</h5>

```

Comment:
Not sure about the `h5` here - it's not semantically correct.

Please use a paragraph, or simply a `span`/`div` and give it appropriate sizing with css.
FWIW I think an `h3` would have been acceptable here, as it would have matched the heading levels in the other sections and meant that we're not jumping around much. But ultimately this is not a heading of a section, but a piece of text that describes the notification, so using an `h` at all is probably not right.

Check out https://www.w3.org/WAI/tutorials/page-structure/headings/ for a bit more info if you want to dig deeper!